packages/subscription-manager/src/components/TabsSwitcher/styles.scss

```diff
@@ -67,3 +67,43 @@
 		margin-bottom: 22px;
 	}
 }
+
+@media screen and ( max-width: $break-medium ) {
```

Comment:
There is a property on `SectionNav` called `allowDropdown` (file: `packages/subscription-manager/src/components/TabsSwitcher/Tabs.tsx`), that may be a built-in solution to disable the dropdown on mobile.

Reply:
I tried that, but I think that wasn't the expected output when that prop was created, as it doesn't work like that.

Reply:
Got it, thanks for clarifying!